frontend/src/pages/projects/screens/projects/useProjectTableRowItems.ts

```diff
@@ -0,0 +1,53 @@
+import { useNavigate } from 'react-router-dom';
+import { useAccessReview } from '~/api';
+import { AccessReviewResourceAttributes, ProjectKind } from '~/k8sTypes';
+
+type Item = {
+  title: string;
+  isDisabled?: boolean;
+  onClick: () => void;
+};
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'rbac.authorization.k8s.io',
+  resource: 'rolebindings',
+  verb: 'create',
+};
+const useProjectTableRowItems = (
+  project: ProjectKind,
+  isRefreshing: boolean,
+  setEditData: (data: ProjectKind) => void,
+  setDeleteData: (data: ProjectKind) => void,
+) => {
+  const [allowCreate] = useAccessReview({
+    ...accessReviewResource,
+    namespace: project.metadata.name,
+  });
+  const navigate = useNavigate();
+  const item: Item[] = [
+    ...(allowCreate
+      ? [
+          {
+            title: 'Edit permissions',
+            onClick: () => {
+              navigate(`/projects/${project.metadata.name}`, { state: 'Permissions' });
+            },
+          },
+        ]
+      : []),
+    {
+      title: 'Edit project',
+      isDisabled: isRefreshing,
+      onClick: () => {
+        setEditData(project);
+      },
+    },
```

Comment:
@kywalker-rh what's the order here?

```
Edit permissions
Edit project
Delete project

// or

Edit project
Edit permissions
Delete project
```
Also, do we want a divider between the edits and delete now that there are multiple edits?

Reply:
Since Edit permissions is going to take the user to another page, instead of opening a modal, let's put Edit project first, and put a divider between the edit actions and delete. So it will be:

Edit project
Edit permissions
----------
Delete project

Otherwise this looks good to me

Reply:
I have made the changes and it looks like this @andrewballantyne @kywalker-rh does this look fine?
  
![editpr](https://github.com/opendatahub-io/odh-dashboard/assets/99238909/c1ef02a0-d18c-4716-92d9-d85bb9798590)

Reply:
Perfect @pnaik1 can you update the screenshots in the description with this image?